File: python/StudentManagement/src/Student.py
class Student:
    def __init__(self, studentId, studentName, studentEmail):
        self.studentId = studentId
        self.studentName = studentName
        self.studentEmail = studentEmail

    def getStudentId(self):
        return self.studentId

    def setStudentId(self, studentId):
        self.studentId = studentId

    def getStudentName(self):
        return self.studentName

    def setStudentName(self, studentName):
        self.studentName = studentName

    def getStudentEmail(self):
        return self.studentEmail

    def setStudentEmail(self, studentEmail):
        self.studentEmail = studentEmail

    def __str__(self):
        return f"Student[studentId={self.studentId}, studentName='{self.studentName}', studentEmail='{self.studentEmail}']"

File: Week01/java/Week01/Student.java
public class Student {
    private int studentId;
    private String studentName;
    private String studentEmail;

    public Student(int studentId, String studentName, String studentEmail) {
        this.studentId = studentId;
        this.studentName = studentName;
        this.studentEmail = studentEmail;
    }

    public int getStudentId() {
        return this.studentId;
    }

    public void setStudentId(int studentId) {
        this.studentId = studentId;
    }

    public String getStudentName() {
        return this.studentName;
    }

    public void setStudentName(String studentName) {
        this.studentName = studentName;
    }

    public String getStudentEmail() {
        return this.studentEmail;
    }

    public void setStudentEmail(String studentEmail) {
        this.studentEmail = studentEmail;
    }

    @Override
    public String toString() {
        return "Student[" +
                "studentId=" + studentId +
                ", studentName='" + studentName + '\'' +
                ", studentEmail='" + studentEmail + '\'' +
                ']';
    }
}